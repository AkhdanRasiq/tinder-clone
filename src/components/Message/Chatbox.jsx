import React from 'react'

class Chatbox extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      strTest: ""
    }
  }

  render() {
    return(
      <div>
        
      </div>
    )
  }
}

export default Chatbox
