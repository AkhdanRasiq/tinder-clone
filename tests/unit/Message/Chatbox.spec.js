import React from 'react';
import { render } from "@testing-library/react"
import { shallow, mount } from "enzyme"
// import { assert } from 'chai'

import Chatbox from "../../../src/components/Message/Chatbox.jsx"

describe("Chatbox.jsx", () => {
  it("Render Chatbox.jsx", async () => {
    const wrapper = shallow(
      <Chatbox
        a_messages={{
          text      : "Test",
          user      : "Admin",
          timestamp : 1643468925
        }}
      />
    )
    await expect(wrapper).toMatchSnapshot();
    const p_chatboxUser = wrapper.find('.chatbox_user')
    await expect(p_chatboxUser.length).toBe(1)
  })
})
