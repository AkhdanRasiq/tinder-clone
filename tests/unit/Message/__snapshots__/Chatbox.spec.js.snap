// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chatbox.jsx Render Chatbox.jsx 1`] = `ShallowWrapper {}`;
